docs/reference/configuration.md

```diff
@@ -32,7 +32,7 @@ class MyApp : android.app.Application {
 }
 ```
 
-1. This will be the name used by {{kib}} when listing your application on the [Services](https://www.elastic.co/guide/en/observability/current/apm-services.html) page. Defaults to `unknown`. See [why your app is referred to as a "service"](faq.md#why-service).
+1. This will be the name used by {{kib}} when listing your application on the [Services](https://www.elastic.co/docs/solutions/observability/apm/services) page. Defaults to `unknown`. See [why your app is referred to as a "service"](faq.md#why-service).
```

Comment:
```suggestion
1. This will be the name used by {{kib}} when listing your application on the [Services](docs-content://solutions/observability/apm/services.md) page. Defaults to `unknown`. See [why your app is referred to as a "service"](faq.md#why-service).
```